script.rb

```diff
@@ -1,20 +1,28 @@
 require 'pry'
+require 'sqlite3'
 require_relative './crawler'
-require_relative './document_collection'
-require_relative './breadth_first_url_collection'
-require_relative './depth_first_url_collection'
 
-BASE_URL = 'http://dragonage.wikia.com'
+db = SQLite3::Database.new 'webcrawler.db'
 
-document_collection = DocumentCollection.new
-url_collection = BreadthFirstUrlCollection.new
-url_collection.add_url('')
-crawler = Crawler.new(document_collection, url_collection)
-
-crawler.run(2)
-puts document_collection.size
-puts url_collection.size
+db.execute <<-SQL
+  CREATE TABLE IF NOT EXISTS URLS_to_crawl (
+    url varchar(200) PRIMARY KEY,
+    state varchar(25)
+  );
+SQL
 
+db.execute("INSERT OR IGNORE INTO URLS_to_crawl (url, state) VALUES ('http://dragonage.wikia.com', 'uncrawled')")
 
+db.execute <<-SQL
+  CREATE TABLE IF NOT EXISTS documents (
+    url varchar(200) PRIMARY KEY,
+    content varchar(65535)
```

Comment:
I agree with TEXT here, it's more commonly used. If you wanted to check the differences between varchar and TEXT, I support that study!